spec/graphql/mutations/login_user_spec.rb

```diff
@@ -58,4 +55,24 @@
       end
     end
   end
+
+  context 'with revoked membership' do
+    let(:revoked_membership) { create(:membership, status: :revoked) }
+
+    it 'returns an error' do
+      result = execute_graphql(
+        query: mutation,
+        variables: {
+          input: {
+            email: revoked_membership.user.email,
+            password: 'ILoveLago',
+          }
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.